src/main/java/io/confluent/connect/jdbc/BulkTableQuerier.java

```diff
@@ -78,6 +78,15 @@ public SourceRecord extractRecord() throws SQLException {
       default:
         throw new ConnectException("Unexpected query mode: " + mode);
     }
+
+    if (keyColumn != null && keySchema != null) {
```

Comment:
Check for both `keyColumn` and `keySchema` seems like it shouldn't be necessary -- if `keyColumn` isn't null, shouldn't we have thrown an exception earlier if we couldn't create the `keySchema`?